lib/bolt/config/transport/local.rb

```diff
@@ -7,22 +7,14 @@ module Bolt
   class Config
     module Transport
       class Local < Base
-        OPTIONS = %w[
-          cleanup
-          interpreters
-          run-as
-          run-as-command
-          sudo-executable
-          sudo-password
-          tmpdir
-        ].freeze
-
         WINDOWS_OPTIONS = %w[
           cleanup
           interpreters
           tmpdir
         ].freeze
 
+        OPTIONS = WINDOWS_OPTIONS.dup.concat(RUN_AS_OPTIONS).freeze
```

Comment:
Concatenating the arrays has the side effect of listing the options out of order in the reference docs. I think there are a couple options here:

1. Sort the array after concatenating
1. Sorting the array before slicing the `OPTIONS` hash in the rake task
1. Don't concatenate the run-as options and just have the array in the base class that is only used by the rake task for the privilege escalation doc

I personally lean towards 3, just because it makes the code a bit easier to follow since you don't need to reference the `RUN_AS_OPTIONS` array when seeing what options are available and we don't group other options like this... but I don't feel super strongly about it.